.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,75 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0,3' # 2/weekly
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
+        platform: [x64]
+    env:
+      PKG_NAME: gensim
+      REPO_DIR: gensim
+      BUILD_COMMIT: HEAD
+      PLAT: x86_64
+      UNICODE_WIDTH: 32
+      MB_PYTHON_VERSION: ${{ matrix.python-version }} # MB_PYTHON_VERSION is needed by Multibuild
+      TEST_DEPENDS: Morfessor==2.0.2a4 python-levenshtein==0.12.0 visdom==0.1.8.9 pytest mock cython nmslib pyemd testfixtures scikit-learn pyemd
+      DOCKER_TEST_IMAGE: multibuild/xenial_x86_64
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Setup OSX var for Multibuild
+      if: ${{ matrix.os == 'macos-latest' }}
+      run: |
+        echo "TRAVIS_OS_NAME=osx" >> $GITHUB_ENV;
+    - name: Setup SKIP_NETWORK_TEST env variable
```

Comment:
Same as above.

Having these variables defined in the matrix is better, because then we can see what's going on up front, instead of having to read the build script step by step.